src/Nox.Integration/Nox.Integration.csproj

```diff
@@ -38,6 +38,10 @@
 
     <ItemGroup>
       <PackageReference Include="ETLBox" Version="3.0.2" />
+      <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```

Comment:
Do we need the Http dependency? this could be used in a class Library?

Reply:
Microsoft.AspNetCore.Http.Abstractions is used to refer on IApplicationBuilder

Reply:
Create an Extension to IServiceColletion would be enough? are we using anything that comes from IApplicationBuilder?